frontend/talentsearch/src/js/components/GovernmentInfoForm/GovernmentInfoForm.tsx

```diff
@@ -263,39 +257,6 @@ export const GovernmentInfoForm: React.FunctionComponent<
           </div>
         </>
       )}
-      {govEmployee === "yes" &&
-        (govEmployeeStatus === GovEmployeeType.Term ||
-          govEmployeeStatus === GovEmployeeType.Indeterminate) && (
-          <p>
-            {intl.formatMessage({
-              defaultMessage:
-                "Please indicate if you are interested in lateral deployment or secondment. Learn more about this.",
-              description:
-                "Text blurb, asking about interest in deployment/secondment in the government info form",
-            })}
-          </p>
-        )}
-      {govEmployee === "yes" &&
-        (govEmployeeStatus === GovEmployeeType.Term ||
-          govEmployeeStatus === GovEmployeeType.Indeterminate) && (
-          <div data-h2-padding="b(bottom, s)">
-            <Checkbox
-              id="lateralDeployBool"
-              label={intl.formatMessage({
-                defaultMessage:
-                  "I am interested in lateral deployment or secondment.",
-                description: "Label displayed on lateral/secondment checkbox",
-              })}
-              name="lateralDeployBool"
-              boundingBox
-              boundingBoxLabel={intl.formatMessage({
```

Comment:
Should the corresponding string in fr.json be removed as well for the `intl.formatMessage` removed?

Reply:
removed with [remove additional secondment fr strings](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3698/commits/634cdf2ed537b2ae7b6bca03521808aa3bb97285)